modules/nlu/src/backend/tools/make-tokens.ts

```diff
@@ -0,0 +1,31 @@
+import _ from 'lodash'
+
+import { Token } from '../typings'
+import { sanitize } from '../pipelines/language/sanitizer'
+
```

Comment:
When using native function such as reduce you don't have to use lodash, unless you want to chain it with non native functions such as groupby, etc. In your case you don't have to. It should look like this instead:
```js
export const makeTokenObjects = (stringTokens: string[], text: string) => stringTokens.reduce((ac, cu) => reduceTokens(ac, cu, text), [] as Token[]))
```
Also, we can see that reduceTokens has almost the same signature as the acual reduce function. The only difference is the dependency to text. If we define it first and return a function with the same signature it would be even better! Like that:

```js
const reduceTokens = (text: string) => (currentTokens: Token[], token: string) => ...
```

Then `makeTokenObjects` becomes:
```js
export const makeTokenObjects = (stringTokens: string[], text: string) => 
stringTokens.reduce(reduceTokens(text), [] as Token[])
```